### ISSUE TITLE
`volta completions zsh` panics

### ISSUE THREAD

**Issue by uasi** [volta-cli/volta#745]
```
% volta --version
0.8.0

% volta completions zsh
thread 'main' panicked at 'Fatal internal error. Please consider filing a bug report at https://github.com/clap-rs/clap/issues', /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/completions/zsh.rs:350:29
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

# On v0.8.0 tag
% VOLTA_LOGLEVEL=debug RUST_BACKTRACE=full cargo run --bin=volta -- completions zsh
    Finished dev [unoptimized + debuginfo] target(s) in 0.20s
     Running `target/debug/volta completions zsh`
thread 'main' panicked at 'Fatal internal error. Please consider filing a bug report at https://github.com/clap-rs/clap/issues', /Users/uasi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/completions/zsh.rs:350:29
stack backtrace:
   0:        0x10f450bbf - backtrace::backtrace::libunwind::trace::hfff3154c85fa162b
                               at /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/libunwind.rs:86
   1:        0x10f450bbf - backtrace::backtrace::trace_unsynchronized::h4a9920525cd627b4
                               at /Users/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.44/src/backtrace/mod.rs:66
   2:        0x10f450bbf - std::sys_common::backtrace::_print_fmt::h972e000d0e4ed1fa
                               at src/libstd/sys_common/backtrace.rs:78
   3:        0x10f450bbf - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hdfed66b3e2e8ce60
                               at src/libstd/sys_common/backtrace.rs:59
   4:        0x10f476f9e - core::fmt::write::h9de7b13cfaa80b03
                               at src/libcore/fmt/mod.rs:1063
   5:        0x10f448d17 - std::io::Write::write_fmt::h9197012b9d65a9a3
                               at src/libstd/io/mod.rs:1426
   6:        0x10f45329a - std::sys_common::backtrace::_print::hb8e24d3c51344086
                               at src/libstd/sys_common/backtrace.rs:62
   7:        0x10f45329a - std::sys_common::backtrace::print::h93407cb3f8a6ecb3
                               at src/libstd/sys_common/backtrace.rs:49
   8:        0x10f45329a - std::panicking::default_hook::{{closure}}::h7b209cdc44c5777f
                               at src/libstd/panicking.rs:204
   9:        0x10f452fdc - std::panicking::default_hook::hf135bc502e6f77db
                               at src/libstd/panicking.rs:224
  10:        0x10f453908 - std::panicking::rust_panic_with_hook::h7967810bc33e523b
                               at src/libstd/panicking.rs:470
  11:        0x10f4534d2 - rust_begin_unwind
                               at src/libstd/panicking.rs:378
  12:        0x10f48708f - std::panicking::begin_panic::hc31d5034796a8de1
  13:        0x10f486f1a - std::panicking::begin_panic::hc31d5034796a8de1
  14:        0x10f3556e8 - core::option::Option<T>::expect::h154606b57accdb43
                               at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/libcore/option.rs:347
  15:        0x10f3cdde6 - clap::completions::zsh::write_opts_of::h9e42f97bad3ccaf7
  16:        0x10f3ccf01 - clap::completions::zsh::parser_of::h3de45a458f999529
  17:        0x10f3cc676 - clap::completions::zsh::subcommands_of::add_sc::h8a12781246690057
  18:        0x10ea9186b - clap::completions::zsh::ZshGen::generate_to::h33edea4ba889d488
                               at /Users/uasi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/completions/zsh.rs:57
  19:        0x10ea72099 - clap::completions::ComplGen::generate::h8a49f33e6422951c
                               at /Users/uasi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/completions/mod.rs:36
  20:        0x10ea8a914 - clap::app::parser::Parser::gen_completions_to::h25ca6a1392766199
                               at /Users/uasi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/app/parser.rs:114
  21:        0x10eaca9bd - clap::app::App::gen_completions_to::h9c81aabb8bbbdde4
                               at /Users/uasi/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.0/src/app/mod.rs:1434
  22:        0x10eacd55b - <volta::command::completions::Completions as volta::command::Command>::run::he9a4d75f37be12b1
                               at src/command/completions.rs:76
  23:        0x10eaa11ff - volta::cli::Subcommand::run::hbeaa8f045723d791
                               at src/cli.rs:140
  24:        0x10eaa0d56 - volta::cli::Volta::run::hf99d43a8fef004a5
                               at src/cli.rs:58
  25:        0x10ead0d3b - volta::main::{{closure}}::h78705481f8ffce83
                               at src/main.rs:30
  26:        0x10eac402a - core::result::Result<T,E>::and_then::h6e8b77aeae8100c2
                               at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/libcore/result.rs:727
  27:        0x10eaa44d2 - volta::main::he227846e86d27c3e
                               at src/main.rs:30
  28:        0x10ea70c3e - std::rt::lang_start::{{closure}}::h7446901196a92a4d
                               at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/libstd/rt.rs:67
  29:        0x10f4533d8 - std::rt::lang_start_internal::{{closure}}::hab79accac7befc7d
                               at src/libstd/rt.rs:52
  30:        0x10f4533d8 - std::panicking::try::do_call::he58e98df0f664580
                               at src/libstd/panicking.rs:303
  31:        0x10f45c0ab - __rust_maybe_catch_panic
                               at src/libpanic_unwind/lib.rs:86
  32:        0x10f453dda - std::panicking::try::h1c15a2595ce4d309
                               at src/libstd/panicking.rs:281
  33:        0x10f453dda - std::panic::catch_unwind::h06858616dd2b84f7
                               at src/libstd/panic.rs:394
  34:        0x10f453dda - std::rt::lang_start_internal::hfa21381d8ff23689
                               at src/libstd/rt.rs:51
  35:        0x10ea70c21 - std::rt::lang_start::h872579ff44b10f85
                               at /rustc/8d69840ab92ea7f4d323420088dd8c9775f180cd/src/libstd/rt.rs:67
  36:        0x10eaa4762 - volta::main::he227846e86d27c3e
```

**Comment by uasi** [comment]
In src/command/run.rs, it is specified that the `--npm` option conflicts with the `--no-npm` option, but there is no `Run::no_npm` field:

https://github.com/volta-cli/volta/blob/a8186ceb9c0c015730d81054894a5a3690403d7a/src/command/run.rs#L20-L22

Either deleting `conflicts_with = "no_npm"` or adding a `Run::no_npm` field fixes this issue.

**Comment by charlespierce** [comment]
Hi @uasi, thanks for reporting this and excellent detective work! It looks like that was an attribute that was left there but not updated to the correct field (`bundled_npm`). Fixed in #746, will release it on Tuesday!
